engine-rest/engine-rest-openapi/src/main/templates/lib/commons/historic-incident-query-params.ftl

```diff
@@ -0,0 +1,148 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/filter/get-query/index.html -->
+
+<#assign sortByValues = [
+  '"incidentId"',
```

Comment:
According to the Java code for `HistoricIncidentQueryDto`, you can also use `incidentMessage` and `incidentState` for sorting. Maybe this needs to be added in the REST documentation as well.

Reply:
No need to fix it in the REST docs. It seems to be updated there already.